frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.scss

```diff
@@ -6,11 +6,12 @@
 }
 
 .usa-button.ttahub-next-steps__add-step-button {
-  margin-top: 0;
+  margin-top: 2px;
+  margin-bottom: 12px;
 }
```

Comment:
is this not-utility classes because of a CSS specificity thing (just curious)? also, weird that that they aren't using the standard margin sizes, which are generally divisible by 8

Reply:
no -- I think I was just trying to get it to be a pixel-perfect match with the figma prototype, but using `margin-bottom-2` on the button and reverting the styles shown above gets it close enough 🙃 06bb7cbd